nimsuggest/tester.nim

```diff
@@ -315,8 +314,12 @@ proc runTest(filename: string): int =
           answer.add '\L'
         doReport(filename, answer, resp, report)
   finally:
-    inp.writeLine("quit")
-    inp.flush()
+    try:
+      inp.writeLine("quit")
```

Comment:
if child already died, these could raise an IOError os OSError with `EPIPE` (or, before https://github.com/nim-lang/Nim/pull/17752, raise signal SIGPIPE), so it seems correct to protect this with try/except;

this diff is not needed for this PR but it helped during debugging and fixing tests as raise EPIPE would hide the error